clang/include/clang/AST/Expr.h

```diff
@@ -3552,6 +3552,15 @@ class CastExpr : public Expr {
   /// function that it invokes.
   NamedDecl *getConversionFunction() const;
 
+  /// Path through the class hierarchy taken by a `DerivedToBase` or
+  /// `UncheckedDerivedToBase` cast. For each derived-to-base edge in the path,
+  /// the path contains a `CXXBaseSpecifier` for the base class of that edge;
+  /// the entries are ordered from derived class to base class.
```

Comment:
You can see in `CastConsistency` the set of cast kinds that require a base path; it's more than just these two.

Reply:
Thanks for pointing this out! I've fixed this -- WDYT?